src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -241,6 +241,25 @@ public ReadTransformer makePostReadFilterTransformer(){
     }
 
     /**
+     * @return Default size in MB of the cloud prefetch buffer. May be overridden by individual tools.
+     *         The default implementation returns a value (40 MB) that is suitable for tools with a small
+     *         number of large cloud inputs. Tools with large numbers of cloud inputs will likely want to
+     *         override to specify a smaller size.
+     */
+    public int getDefaultCloudPrefetchBufferSize() {
```

Comment:
Shouldn't these be protected since they're really internal methods designed for overriding?